scripts/mypy/requirements.txt

```diff
@@ -0,0 +1,8 @@
+mypy==0.971
+pydantic[email]==1.9.2
+types-aiofiles==0.8.10
```

Comment:
why are some of those fixed?

Reply:
this was taken from osparc-simcore where  is used in CI. Therefore I guess this is to guarantee repeatibility.
Here we can leave unconstraint fro the moment.